src/server/frontend/wayland/wl_keybaord.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2018 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Christopher James Halse Rogers <christopher.halse.rogers@canonical.com>
+ */
+
+#ifndef WL_KEYBOARD_H
```

Comment:
I may have missed this elsewhere, but we start our include guards with "MIR_", in this case it should be "MIR_FRONTEND_WL_KEYBOARD_H_"

Reply:
I'll fix that.

Reply:
hold on, I did it wrong.

Reply:
fixed.